src/collections/_documentation/platforms/node/express.md

```diff
@@ -26,6 +26,8 @@ Sentry.init({ dsn: '___PUBLIC_DSN___' });
 // The request handler must be the first middleware on the app
 app.use(Sentry.Handlers.requestHandler());
 
+// All controllers should live here
+
```

Comment:
@twolfson mind putting an example of an app-level middleware (e.g. `app.get()`) to drive the point home?

Reply:
Sure thing!